solutions/vmware/migration/ansible/vm-metadata-operations/roles/install_packages/tasks/install_docker.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Install yum utils
+  yum:
+    name: yum-utils
+    state: latest
+
+- name: Add Docker repo
+  get_url:
+    url: https://download.docker.com/linux/centos/docker-ce.repo
+    dest: /etc/yum.repos.d/docer-ce.repo
+
+- name: Install Docker
+  package:
+    name: docker-ce
+    state: latest
+
+- name: Start Docker service
```

Comment:
@gauravchaudhari02 Can you please add steps to create `/etc/docker/daemon.json`  and stop/start the docker service

Reply:
@shailesh-vaidya , Added requested tasks